src/components/alerting/edit-alerting.js

```diff
@@ -0,0 +1,75 @@
+import { UpdateAlertingConfigModel } from '../../models/alerting';
+import PropTypes from 'prop-types';
+import { TextField } from '@equinor/eds-core-react';
+import * as React from 'react';
```

Comment:
Avoid importing entire components where possible
```suggestion
import { useEffect, useState } from 'react';
```

Reply:
fixed